api/src/messaging/api_views.py

```diff
@@ -33,6 +43,41 @@ def get_queryset(self):
         return self.request.user.rooms.all()
 
 
+class MembershipCreateDestroy(APIView):
+    permission_classes = [
+        IsAuthenticated,
+        IsMember,
+    ]
+
+    @swagger_auto_schema(tags=["Messaging"])
+    def post(self, request, *args, **kwargs):
+        member = request.data
+        roomid = kwargs["id"]
+        room = Room.objects.get(id=roomid)
+        if room.members.filter(id=member):
+            return Response({"message": "Member already exists"})
+
+        try:
+            user = CustomUser.objects.get(id=member)
+            room.members.add(user)
+            return Response({"message": "Member Created."})
```

Comment:
Probably give a better response message, since we didn't actually create a member